data/classes/a.json

```diff
@@ -69,7 +69,17 @@
     },
     {
       "name": "<init>",
-      "descriptor": "(Lb;)V"
+      "descriptor": "(Lb;)V",
+      "javadoc": [
+        "Copy constructor for this class."
+      ],
```

Comment:
This is not the case. This takes a `Matrix4f` and converts it into a `Matrix3f`. This is used to get the resulting transformation or the normal matrix itself.

Reply:
Fixed to document the actual copy constructor.